db/migrate/20190925171050_create_user_blocks.rb

```diff
@@ -0,0 +1,11 @@
+class CreateUserBlocks < ActiveRecord::Migration[5.2]
+  def change
+    create_table :user_blocks do |t|
+      t.bigint :blocked_id, null: false
```

Comment:
Indexes and foreign keys are needed for `blocked_id` and `blocker_id`. A unique index on `blocker_id` + `blocked_id` would be useful too.